projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -67,13 +67,18 @@ public function register_routes() {
 							'description'          => 'Marks a checklist as dismissed by the user',
 							'type'                 => 'object',
 							'properties'           => array(
-								'slug'         => array(
+								'slug'            => array(
 									'description' => 'Checklist slug',
 									'type'        => 'string',
 									'enum'        => $this->get_checklist_slug_enums(),
 								),
-								'is_dismissed' => array(
-									'type' => 'boolean',
+								'is_dismissed'    => array(
+									'type'     => 'boolean',
+									'required' => false,
+									'default'  => false,
+								),
+								'dismissed_until' => array(
+									'type' => 'number',
```

Comment:
I think we should add a description here, so it be more clear what this number means.

Reply:
Field removed